x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.tsx

```diff
@@ -252,6 +325,54 @@ export const HostList = () => {
     ];
   }, [formatUrl, queryParams, search]);
 
+  const renderTableOrEmptyState = useMemo(() => {
+    if (listData && listData.length > 0) {
```

Comment:
TODO, the flow works, but some oddities in loading cause "pop ins", gonna introduce some additional loading states